### PR TITLE
README: Split into Getting Started + Learn More

### PR DESCRIPTION
**Pull request by yondonfu**
A few changes to the README to include initial step-by-step recommendations for getting onboarded to the protocol.

### CODE REVIEW

File: README.md
Comment (chrishobcroft) [inline README.md:1]: What about something about engaging with the project's "build in the open" process? e.g. the LAWG, and other open research, development and operational tracks?

Comment (yondonfu) [reply README.md:1]: Good idea! I'll leave that out of the scope of this PR for now though as my intent is just to add initial step-by-step recommendations for getting onboarded to the protocol and additional recommendations can be considered separately.

**Review by dob — COMMENTED** [review]
It looks like an improvement over what we have, so I'm good with it. But it does leave the wiki readme feeling a little protocol-heavy. If others also think it's too specific for the README, then one option would be that the main readme says something like "For those looking to learn more about the details, development, and Livepeer protocol itself, please view our Protocol Info Page", which links to a separate page containing all the protocol and LIP specific links.

**Review by victorges — APPROVED** [review]
LGTM either way (in home or separate page)

**Comment by yondonfu** [comment]
Merging for now since this is an improvement over the current README, but open to moving content to a separate page whether it be in this repo or in the docs separately if there is a better place there!

**Pull request merged**